src/resources/views/profile/index.blade.php

```diff
@@ -1,9 +1,11 @@
 @extends('layouts.app')
 
-@section('title', 'My Thoughts')
+@section('title', $user->username)
```

Comment:
what if $user->username is null, what will be displayed?

Reply:
i have implemented ternary operator to handle null value